GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -36,6 +37,7 @@ public partial class UserRepositoriesList : GitExtensionsControl
         private Brush _hoverColorBrush = new SolidBrush(SystemColors.InactiveCaption);
         private ListViewItem _prevHoveredItem;
         private readonly ListViewGroup _lvgRecentRepositories;
+        private IReadOnlyList<Repository> _missingRepositories;
```

Comment:
❓ I'm not a fan of trying to maintain this as a local state variable. Is there a way to calculate this value in the click handler instead?

Reply:
I feel the same but I believe there isn't. There is no easy way to access the general repositories list, so I re-calculate the missing repos whenever the general list is being calculated.
I also think it may be too heavy in the click handler.

Reply:
Thanks for the response. I'm going to take a look at this today and try to find another way. If I can't then I'll ✔️ this.